sphinx_gherkindoc/writer.py

```diff
@@ -310,9 +312,17 @@ def table(table: behave.model.Table, inline: bool = False) -> None:
     if AVAILABLE_ROLES:
         output_file.blank_line()
 
-    feature = behave.parser.parse_file(source_path)
+    feature_class = parsers.get(feature_parser, None)
+    if not feature_class:
+        raise KeyError(
```

Comment:
Much later at night reviewing than I expected, but with the use of `choices` in the command line parser, is this still possible?  I'm not sure, so asking here before I forget...

Reply:
It is true that `argparse`'s `choices` arg setup will protect us from getting invalid parser options passed in via the CLI. I considered removing this block, but decided that (a) I couldn't envision any CLI-based situation in which it would be a _problem_, and (b) it could be a useful check if `writer.feature_to_rst` were called programmatically elsewhere.

Reply:
Ah. Agreed on point `(b)`